### PR TITLE
New healthcare

### PR DESCRIPTION
**Pull request by engelke**

### CODE REVIEW

File: healthcare/api-client/datasets/datasets.py [hunk 1]
```diff
@@ -33,9 +33,9 @@ def get_client(service_account_json, api_key):
         service_account_json)
     scoped_credentials = credentials.with_scopes(api_scopes)
 
-    discovery_url = '{}?labels=CHC_ALPHA&version={}&key={}'.format(
+    discovery_url = '{}?labels=CHC_BETA&version={}&key={}'.format(
         discovery_api, api_version, api_key)
-
+
```
Comment (gguuss) [inline healthcare/api-client/datasets/datasets.py]: nit. trailing whitespace

File: healthcare/api-client/dicom/dicom_stores.py [hunk 2]
```diff
@@ -25,22 +25,23 @@ def get_client(service_account_json, api_key):
     """Returns an authorized API client by discovering the Healthcare API and
     creating a service object using the service account credentials JSON."""
     api_scopes = ['https://www.googleapis.com/auth/cloud-platform']
-    api_version = 'v1alpha'
+    api_version = 'v1alpha2'
```
Comment (gguuss) [inline healthcare/api-client/dicom/dicom_stores.py]: Just FYI, I was seeing v1beta1 elsewhere, maybe a typo?

File: healthcare/api-client/dicom/dicom_stores.py [hunk 3]
```diff
@@ -342,14 +415,29 @@ def parse_command_line_args():
...
     command = parser.add_subparsers(dest='command')
 
     command.add_parser('create-dicom-store', help=create_dicom_store.__doc__)
     command.add_parser('delete-dicom-store', help=delete_dicom_store.__doc__)
     command.add_parser('get-dicom-store', help=get_dicom_store.__doc__)
     command.add_parser('list-dicom-stores', help=list_dicom_stores.__doc__)
     command.add_parser('patch-dicom-store', help=patch_dicom_store.__doc__)
-
+    command.add_parser(
+        'get_iam_policy',
```
Comment (gguuss) [inline healthcare/api-client/dicom/dicom_stores.py]: lint is not liking this line

File: healthcare/api-client/datasets/datasets.py (hunk 1 above)
Comment (engelke) [reply healthcare/api-client/datasets/datasets.py]: I just fixed the two lint errors. I'll remove the trailing whitespace ASAP.

File: healthcare/api-client/dicom/dicom_stores.py (hunk 2 above)
Comment (engelke) [reply healthcare/api-client/dicom/dicom_stores.py]: This is a typo, a serious one. I'll fix it after rerunning tests.

File: healthcare/api-client/dicom/dicom_stores.py (hunk 3 above)
Comment (engelke) [reply healthcare/api-client/dicom/dicom_stores.py]: I'll thought I'd handled the lint issues. I'll clean them all up in about an hour.

File: healthcare/api-client/hl7v2/hl7v2_stores.py [hunk 4]
```diff
@@ -25,15 +25,15 @@ def get_client(service_account_json, api_key):
     """Returns an authorized API client by discovering the Healthcare API and
     creating a service object using the service account credentials JSON."""
     api_scopes = ['https://www.googleapis.com/auth/cloud-platform']
-    api_version = 'v1alpha'
+    api_version = 'v1alpha2'
```
Comment (gguuss) [inline healthcare/api-client/hl7v2/hl7v2_stores.py]: should this be alpha or beta?

File: healthcare/api-client/dicom/dicom_stores.py (hunk 2 above)
Comment (gguuss) [reply healthcare/api-client/dicom/dicom_stores.py]: Checking again if this should be alpha or beta

File: healthcare/api-client/hl7v2/hl7v2_stores.py (hunk 4 above)
Comment (engelke) [reply healthcare/api-client/hl7v2/hl7v2_stores.py]: v1beta1

**Pull request merged**